schema/experiment.yml

```diff
@@ -87,10 +87,11 @@ $defs:
   ControlVariant:
```

Comment:
The notion of a "Control Variant" is not entirely clear to me–I think this represents a control, but I do not know how to interpret the concordant field. Some clarifying examples might help.

Reply:
The intention is to represent variants that were used during assay design or validation. The concordance field should describe whether the control variant agrees with the assay result being described here. It seemed like this was much more straightforward than trying to encode the control information itself, since this could be very heterogeneous (another different functional assay, a ClinVar record, etc.).

Does this seem like a reasonable way to approach this problem? My next task is to work through some examples much like we did for the original minimum information standards.